core/graph/src/main/java/uk/gov/gchq/gaffer/graph/hook/NamedViewResolver.java

```diff
@@ -38,20 +39,21 @@
  * A {@link GraphHook} to resolve {@link NamedView}s.
  */
 @JsonPropertyOrder(alphabetic = true)
-public class NamedViewResolver implements GraphHook {
+public class NamedViewResolver implements GraphHook, GetFromCacheHook {
```

Comment:
The class `GetFromCacheHook` extends `GraphHook`, so there's no need to also implement `GraphHook` here.

Reply:
It's redundant, but didn't think verbose was that bad.
Shall I remove?